packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/Sell/template.success.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import styled from 'styled-components'
+import { path } from 'ramda'
+
+import { Remote } from 'blockchain-wallet-v4/src'
+import * as service from 'services/CoinifyService'
+import Stepper, { StepView } from 'components/Utilities/Stepper'
+import OrderCheckout from '../OrderCheckout'
+import { OrderDetails, OrderSubmit } from '../OrderReview'
+import Payment from 'modals/CoinifyExchangeData/Payment'
+import ISignThis from 'modals/CoinifyExchangeData/ISignThis'
+
+const CheckoutWrapper = styled.div`
+  width: 55%;
+`
+const OrderSubmitWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 35%;
+  padding: 30px 30px 30px 10%;
+`
+const FlexRow = styled.div`
+  display: flex;
+  flex-direction: row;
+`
+
+const Sell = props => {
+  const {
+    fetchSellQuote,
+    refreshQuote,
+    sellQuoteR,
+    clearTradeError,
+    currency,
+    rateQuoteR,
+    checkoutBusy,
+    setMax,
+    paymentMedium,
+    initiateSell,
+    step,
+    busy,
+    trade
+  } = props
+
+  const profile = Remote.of(props.value.profile).getOrElse({ _limits: service.mockedLimits, _level: { currency: 'EUR' } })
+
+  const defaultCurrency = currency || 'EUR' // profile._level.currency
+  const symbol = service.currencySymbolMap[defaultCurrency]
+
+  const limits = service.getLimits(profile._limits, defaultCurrency)
+
+  if (step !== 'isx') {
+    return (
+      <Stepper initialStep={0}>
+        <StepView step={0}>
+          <CheckoutWrapper>
+            <OrderCheckout
+              quoteR={sellQuoteR}
+              rateQuoteR={rateQuoteR}
+              onFetchQuote={fetchSellQuote}
+              limits={limits.sell}
+              type={'sell'}
+              reason={'has_remaining'} // placeholder for now - coinify does not require a reason
```

Comment:
Here are a few reasons that we had to handle at at least one point:

https://github.com/blockchain/My-Wallet-V3-Frontend/blob/master/assets/js/services/coinify.service.js#L62-L66